generators/server/templates/src/test/java/package/cucumber/stepdefs/_StepDefs.java

```diff
@@ -2,13 +2,15 @@
 
 import <%=packageName%>.<%= mainClass %>;
 
-import org.springframework.boot.test.SpringApplicationContextLoader;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.web.WebAppConfiguration;
 import org.springframework.test.web.servlet.ResultActions;
 
+import org.springframework.boot.test.context.SpringBootTest;
+
 @WebAppConfiguration
-@ContextConfiguration(classes = <%= mainClass %>.class, loader = SpringApplicationContextLoader.class)
+@SpringBootTest
+@ContextConfiguration(classes = <%= mainClass %>.class)
```

Comment:
@tan9 Looking a bit further into this, I noticed that a lot of other `@SpringBootTest`-annotated classes do _not_ have a `@ContextConfiguration`. It turns out that it is required for this class, though, due to the `@WebAppConfiguration`, see [here](http://docs.spring.io/spring/docs/4.3.6.RELEASE/javadoc-api/org/springframework/test/context/web/WebAppConfiguration.html).